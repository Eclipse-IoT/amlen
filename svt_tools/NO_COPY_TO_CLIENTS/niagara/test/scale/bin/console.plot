137670
175776
136259
137649
137369
137827
137225
137621
137680
137633
137455
136979
137418
174654
137440
137684
137517
137457
137644
137428
137550
137434
137548
138000
137460
174380
137776
137589
137086
137500
137544
137348
137615
137561
137752
138394
137335
174523
137961
137118
137368
137405
137673
137376
137489
137298
137726
137532
137703
174580
137437
137552
137385
137538
136967
137604
137383
138399
137761
137302
