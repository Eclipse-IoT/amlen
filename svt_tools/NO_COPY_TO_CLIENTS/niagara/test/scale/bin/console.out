"subscription1","/topic/#","c0001199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0005124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0007124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0007199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0004123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0006123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0008123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0007123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0001124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0000123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0000199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0004199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0008199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0002124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0001123","True",19999990,19999990,99.99995,20000000,0,0
"subscription1","/topic/#","c0009124","True",20000000,20000000,100.0,20000000,0,0
"subscription1","/topic/#","c0002199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0005199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0009199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0003124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0006124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0003123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0005123","True",19999985,19999985,99.999925,20000000,0,0
"subscription1","/topic/#","c0003199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0010199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0004124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0002123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0008124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0010124","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0006199","True",0,0,0.0,20000000,0,0
"subscription1","/topic/#","c0009123","True",19999925,19999925,99.999625,20000000,0,0
"subscription1","/topic/#","c0010123","True",19999925,19999925,99.999625,20000000,0,0

